library/src/main/java/com/datatorrent/lib/window/Quantification.java

```diff
@@ -0,0 +1,31 @@
+package com.datatorrent.lib.window;
+
+/**
+ * Created by siyuan on 6/3/16.
+ */
+public class Quantification
+{
```

Comment:
I just discussed with Siyuan. We agreed to separate Duration specification and number of tuple specification, and there will be no more Object[] in the rest of the code.  I will make the changes in my local repo.